viewer/.eslintrc.js

```diff
@@ -14,7 +14,8 @@ module.exports = {
     ecmaVersion: 2020, // Allows for the parsing of modern ECMAScript features
     sourceType: 'module' // Allows for the use of imports
   },
-  plugins: ['header', '@typescript-eslint', 'jsdoc'],
+  plugins: ['header', '@typescript-eslint', 'jsdoc', 'unused-imports'],
```

Comment:
Let’s set it as warn. It will be less annoying then, but precommit hook still will catch it

Reply:
The thing is that compile will fail in CI if there are unused imports, @maksnester  - so I actually think it should be an error (if we don't change that in test compile).